src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -440,20 +440,19 @@ applyUpgrades v cid height
       txs <- map (fmap payloadObj) <$> liftIO txsIO
 
       --
-      -- Note (emily): the historical use of 'mapM_' here means that we are not
-      -- threading the updated module cache from tx to tx in our upgrades.
-      -- This means that the outputed module cache is the set of modules
-      -- loaded in the /last/ tx, and that result will go into the hash.
+      -- Note (emily): the historical use of 'mapM' here means that we are not
```

Comment:
This first paragraph is no longer relevant, just keep your comment starting on L448